layouts/partials/tw/internal-page-nav.html

```diff
@@ -0,0 +1,60 @@
+<div
+  class="mx-auto flex max-w-screen-xl  flex-col md:flex-row"
+  data-ga-category="sticky"
+>
+  <div class="no-js:hidden relative mx-auto w-96 max-w-full md:hidden">
+    <select
+      x-data
+      @change="
+        window.location.assign($event.target.value);
+        $report({
```

Comment:
I think $report is broken with the current analytics setup. Just remove this part and leave the `window.location.assign($event.target.value);` part.